src/python/WMComponent/AnalyticsDataCollector/AnalyticsPoller.py

```diff
@@ -142,10 +144,18 @@ def algorithm(self, parameters):
 
             self.localSummaryCouchDB.uploadData(requestDocs)
             logging.info("Request data upload success\n %s request, \nsleep for next cycle" % len(requestDocs))
-            DataUploadTime.setInfo(uploadTime, "ok")
+
+            self.centralWMStatsCouchDB.updateAgentInfoInPlace(self.agentInfo["agent_url"],
+                                                              {"data_last_update": uploadTime, "data_error": "ok"})
 
         except Exception as ex:
             logging.error("Error occurred, will retry later:")
```

Comment:
How about replacing these error logging by one exception? traceback import can probably be removed then.

Reply:
Thanks, I will change that.